src/ShadowDOM/wrappers/Selection.js

```diff
@@ -56,6 +53,15 @@
     }
   };
 
+  // Not all browsers support Selection.extend. Some versions of IE do not support extend, code that checks
```

Comment:
I know IE11 on windows does not have it. That is what I am testing with. It seems also some one already discovered this and checks if IE is running and not test extend. I changed the test.